au/code/au/unit_of_measure.hh

```diff
@@ -1150,18 +1150,18 @@ struct UnitAvoidance<CommonUnit<Us...>> : std::integral_constant<int, 6> {};
 
 template <typename... Us>
 struct UnitAvoidance<CommonPointUnit<Us...>> : std::integral_constant<int, 7> {};
-}  // namespace detail
+}  // namespace auimpl
 
 template <typename A, typename B>
 struct InOrderFor<UnitProduct, A, B>
     : LexicographicTotalOrdering<A,
                                  B,
-                                 detail::OrderByUnitAvoidance,
-                                 detail::OrderByDim,
-                                 detail::OrderByMag,
-                                 detail::OrderByScaleFactor,
-                                 detail::OrderByOrigin,
-                                 detail::OrderAsUnitProduct,
-                                 detail::OrderAsOriginDisplacementUnit> {};
+                                 auimpl::OrderByUnitAvoidance,
```

Comment:
I think I'm going to skip doing this for now, because `au::detail::UnitAvoidance` is actually in our public facing documentation.  We'll need to give folks a migration path away from naming the `detail` namespace before we change its name.